collab-database/src/blocks/block.rs

```diff
@@ -122,11 +123,15 @@ impl Block {
     }
 
     let encoded_collab = default_database_row_data(&row_id, row);
-    let collab = self
+    let mut collab = self
       .create_collab_for_row(&row_id, Some(encoded_collab))
       .await?;
+    let _ = collab.data.insert_with_path(
```

Comment:
there is a row id in meta
<img width="808" alt="image" src="https://github.com/user-attachments/assets/792daba2-e42d-44f4-b7bf-9bcc9eae27ab">